src/index.html

```diff
@@ -3,9 +3,31 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>Leaderboard</title>
 </head>
 <body>
-    <h1>Hello Anvi !</h1>
+    <h1>Leaderboard</h1>
+    <div class="container">
+        <div class="score">
+            <div class="top-title">
+                <h2 class="title">Recent Score</h2>
+                <button class="refresh" type="reset">Refresh</button>
+            </div>
+            <ul class="board">
+                <li class="user-score">Name: 30</li>
+                <li class="user-score">Name: 40</li>
+                <li class="user-score">Name: 50</li>
+                <li class="user-score">Name: 60</li>
+                <li class="user-score">Name: 70</li>
+            </ul>
+        </div>
```

Comment:
- [ ] Please move the **`index.html`** into the **`index.js`**, and use the best practice for ES6